docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,241 @@
+# LORIS Developers Guide
+This document covers topics including making PRs, issues, and how to go through a normal LORIS workflow when contributing to the LORIS repository. Many notes in this guide are also relevant to contributing code on a fork or instance of LORIS.
+
+## Links to know, read, and follow
+* All markdowns in the Loris GitHub under [docs/](../..) 
+* [Contributing.md](../../../CONTRIBUTING.md) - How to contribute to Loris
+* [Coding Standards](../../CodingStandards.md)
+* [Pull Request Checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist) and [Code Review guidelines](Code_Review_Guide.md)
+
+## 1. Working off your fork
+During installation, you made your own fork of LORIS. Now you can add two remote options in order to switch between your fork and “aces”.
+
+    git remote rename origin aces
+    git remote add my_fork <fork_url>
+             
+Your fork url will usually have the format:  
+    `https://github.com/github-username/LORIS`  
+    
+Any development should be done on your fork of LORIS, and **not** on aces/LORIS. 
+Before starting any PR, be mindful of which branch you want to base off of (main vs release-specific branch).   
+
+You will make a different branch for each PR or change that you are making.  
+
+Here is a basic workflow if you are trying to make a PR off of the _main_ branch:
+
+    git fetch aces
+    git checkout aces/main 
+    git checkout -b "branch_name"
+    vi filename
+    git add filename 
+    git status 
+    git commit -m "Initial commit"
+    git push my_fork branch_name
+
+There is usually no need to fully update your fork to the newest aces/LORIS version, since you can work off whichever branch you just fetched and checked-out.
+
+ **Never** push to aces/Loris. Your branch will be deleted. Always push to your fork, and then create the PR from GitHub.
+
+#### Branch Naming
+Follow the convention most developers use: “date-title...”.   
+Starting with the date allows branches to sort and display in order, and also helps sort out any conflicts faster in the main Loris repository.
+Don’t forget to make your branch names short and informative, like your commit messages, since they’re read by others on GitHub. 
+
+### Helpful commands to run before pushing new code:
+These commands catch small formatting errors that will cause Travis to fail.
```

Comment:
```suggestion
These commands catch small formatting errors that will cause Travis CI (integrated on our GitHub) to fail.
```

Reply:
@AlexandraLivadas both here and in the next section talk about not tripping up travis, but Travis is only introduced in the next section. Could you change the headers / intro sentence so that this flows better?